tests/unit/modules/test_network.py

```diff
@@ -228,6 +225,36 @@ def test_ip_addrs6(self):
         ):
             self.assertListEqual(network.ip_addrs6("int", "include"), ["A"])
 
+    def test_ip_addrs6_type(self):
+        """
+        Test for returns a list of IPv6 addresses assigned to the host filtered by type.
+        """
+        ipv6_addrs = ["fe80::f03c:91ff:fef1:21e3", "2600:3c03::f03c:91ff:fef1:21e3"]
```

Comment:
I grabbed these examples from a random old machine of mine as python's IP address validation did not like purely example addresses that I had tried